iocage_lib/ioc_fetch.py

```diff
@@ -944,6 +946,7 @@ def fetch_update(self, cli=False, uuid=None):
                         )
 
             finally:
+                su.Popen(['umount', f'{mount_root}/dev']).communicate()
```

Comment:
Wont this try to umount twice if you hit the exceptions?

Reply:
No, `EXCEPTION` level means that we will exit this call for sure the first time it occurs, that's why the need to have these on multiple places

Reply:
Are you sure? finally should run even if you raise an exception.

Reply:
Oh, okay the above one is unintended :) Good catch. I was thinking that we only have 2 umount commands - should've looked properly.